debuginfo/src/elf.rs

```diff
@@ -234,7 +234,8 @@ impl<'d> ElfObject<'d> {
     /// [`has_debug_info`](struct.ElfObject.html#method.has_debug_info).
     pub fn debug_session(&self) -> Result<DwarfDebugSession<'d>, DwarfError> {
         let symbols = self.symbol_map();
-        DwarfDebugSession::parse(self, symbols, self.load_address())
+        let relocatable = self.kind() == ObjectKind::Relocatable;
```

Comment:
I would prefer we pass the entire object kind instead of starting to pass bools. I generally regretting every bool passed as argument after a while.